docs/reference/project-configuration.md

```diff
@@ -24,6 +24,6 @@ The file must be called `spatialos.json` and must have the following structure:
 | :------------- | :------------- | :------- | 
 | `configurationVersion`  | Required  | Specifies the version of the configuration file itself. This documentation is for configuration version `0.1` and this field should always be set to this value if you intend to use features described here. |
 | `projectName`  | Required  | The name of the project assigned to you by Improbable. It's used in the Console and for deployments. It should be between 3 and 32 characters long and contain letters, numbers and underscores. |
-| `schemaDescriptor` | Required | A relative (to the location of this file) or absolute path to the SpatialOS schema descriptor file. This is a binary file obtained by using the SpatialOS schema compiler on all of the project's `.schema` files. See the example in the `build.sh` to see how this file can be generated. <br> <br> This file is required in order for the SpatialOS Runtime to understand the schema used by all the workers. |
+| `schemaDescriptor` | Required | A relative (to the location of this file) or absolute path to the SpatialOS schema descriptor file. This is a binary file obtained by using the SpatialOS schema compiler on all of the project's `.schema` files. See the example in the `build.sh` to see how this file can be generated. <br><br> This file is required in order for the SpatialOS Runtime to understand the schema used by all the workers. |
```

Comment:
```suggestion
| `schemaDescriptor` | Required | A relative (to the location of this file) or absolute path to the SpatialOS schema descriptor file. This is a binary file obtained by using the SpatialOS schema compiler on all of the project's `.schema` files. See the example in the `build.sh` of how to generate this file. <br><br> This file is required in order for the SpatialOS Runtime to understand the schema used by all the workers. |
```

Minor change - do change it back if what I've done makes no sense.

Reply:
I rephrased the sentence to match your idea.